app/controllers/admin/volunteers.py

```diff
@@ -97,3 +98,12 @@ def removeVolunteerFromEvent(user, eventID):
     (EventRsvp.delete().where(EventRsvp.user==user)).execute()
     flash("Volunteer successfully removed", "success")
     return ""
+
+@admin_bp.route('/updateBackgroundCheck', methods = ['POST'])
+def updateBackgroundCheck():
+    eventData = request.form
```

Comment:
Only CELTS admins (and not student staff) should be able to change the background check status

Reply:
We have made changes to this but the UI should do not allow them to make changes if they are not an admin.